File: csharp/LeetTests/Old/RepeatedDnaSequencesTest.cs
using Leet.RepeatedDnaSequences;

namespace LeetTests.Old
{
    internal class RepeatedDnaSequencesTest
    {
        [Test]
        public void TestsRepeatedDnaSequences()
        {
            var soltuion = new Solution();

            Assert.That(
                soltuion.FindRepeatedDnaSequences("AAAAACCCCCAAAAACCCCCCAAAAAGGGTTT"),
                Is.EqualTo(new[] { "AAAAACCCCC", "CCCCCAAAAA" })
            );

            Assert.That(
                soltuion.FindRepeatedDnaSequences("AAAAAAAAAAAAA"),
                Is.EqualTo(new[] { "AAAAAAAAAA" })
            );

            Assert.That(
                soltuion.FindRepeatedDnaSequences("AAAAAAAAAAA"),
                Is.EqualTo(new[] { "AAAAAAAAAA" })
            );
        }
    }
}

File: python/today15/repeated_dna_sequences_test.py
from repeated_dna_sequences import Solution

def test1():
    result = Solution().findRepeatedDnaSequences("AAAAACCCCCAAAAACCCCCCAAAAAGGGTTT")
    assert result == ["AAAAACCCCC","CCCCCAAAAA"]

def test2():
    result = Solution().findRepeatedDnaSequences("AAAAAAAAAAAAA")
    assert result == ["AAAAAAAAAA"]

def test3():
    result = Solution().findRepeatedDnaSequences("AAAAAAAAAAA")
    assert result == ["AAAAAAAAAA"]